test/test.config.js

```diff
@@ -16,15 +16,15 @@ if(process.env.MONGODB_USERNAME && process.env.MONGODB_PASSWORD) {
   config.mongodb.username = process.env.MONGODB_USERNAME;
   config.mongodb.password = process.env.MONGODB_PASSWORD;
   const {connectOptions} = config.mongodb;
-  // this can be false
-  connectOptions.ssl = true;
+  // process.env.MONGODB_SSL should be 1 to test with SSL
+  connectOptions.ssl = Boolean(process.env.MONGODB_SSL) || false;
```

Comment:
As long as we're ensuring the tests run against SSL-enabled servers, this is fine.

Reply:
that is another issue actually (which we will need to address with changes to the docker images used to run mongodb servers). You have to specify a command when starting the mongoDB server with a valid SSL cert in order to run mongodb in SSL mode. I'm not sure when that changed from a minor error to a fatal error, but as of `mongodb 4.4` it no longer allows you to login if the server doesn't have a valid SSL certificate and SSL is required. This is fixable quite easily by passing a let's cert certificate to the mongodb docker image and then add the ssl cli command with a path to the certificate on start up. We don't do that currently which means projects that use docker mongodb images are probably not running with SSL. We also might want to add one of those change this for production guards to ensure in production we're running with SSL enabled. Basically good idea, but probably better address in a separate PR/ issue.